index.html

```diff
@@ -105,12 +105,13 @@ <h4>Pitävätkö laskurin hinnat varmasti paikkansa?</h4>
                 <hr>
                 <h3>Päivitykset</h3>
                 <p>
-                    <li>9.1.2020 klo 7.30: korjattu R-kioskin prosenttiosuuden laskeminen siten, että
+                    <li>14.1.2021: Päivitetty HSL:n uudet kausilippujen hinnat laskuriin.</li>
```

Comment:
```suggestion
                    <li>14.1.2021 klo 20.30: Päivitetty HSL:n uudet kausilippujen hinnat laskuriin.</li>
```
Tähän voisi lisätä kellonajan